student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(age, classroom, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission)
```

Comment:
- [ ] The parent_permission should be set as required considering it's declaration with ":".

  ```
  super(age, name, parent_permission: parent_permission)
  ```